css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,99 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+  /* Keep the header bar visible when the sidebar is open */
+  height: calc(100vh - var(--layout-header-height-mobile));
+  margin-top: var(--layout-header-height-mobile);
+  padding-bottom: calc(var(--layout-header-height-mobile) / 2);
+
+  display: flex;
```

Comment:
It seemed to me that only the `top` CSS property is overridden by inline styles, which is why I used `margin-top` instead. If also `margin-top` is going to be overridden, then we'll need to completely re-discuss how we're going to achieve the layout required by our template design.